buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -664,39 +671,45 @@ def set_test_targets() {
 * Set EXTRA_TEST_LABELS map if defined in variable file.
 */
 def set_test_misc() {
-    EXTRA_TEST_LABELS = [:]
     if (!params.TEST_NODE) {
         // Only add extra test labels if the user has not specified a specific TEST_NODE
-        TARGET_NAMES.each { target ->
-            EXTRA_TEST_LABELS[target] = buildspec.getVectorField("extra_test_labels", target).join('&&') ?: ''
+        TESTS.each { id, target ->
+            target['extraTestLabels'] = buildspec.getVectorField("extra_test_labels", target).join('&&') ?: ''
         }
     }
 
-    EXCLUDED_TESTS = []
     def buildspec = buildspec_manager.getSpec(SPEC)
     def excludedTests = buildspec.getVectorField("excluded_tests", SDK_VERSION)
     if (excludedTests) {
-        EXCLUDED_TESTS.addAll(excludedTests)
+        excludedTestsArray = []
```

Comment:
`excludedTestsArray` is not required since it contains `excludedTests ` collection only and is used only in this block.